### ISSUE TITLE
Multiple pen colors in same page.

### ISSUE THREAD

**Issue by saurabhdaware** [saurabhdaware/text-to-handwriting#68]
A lot of people reached out to me on mails and Twitter to have this feature.

We can keep everything the way it is right now, but we can also allow users to select text and then change the value in drop-down to color only the selected part.

**Comment by Himanshunitrr** [comment]
To color the font or the background sir.Will like to work on this too.

**Comment by saurabhdaware** [comment]
https://github.com/saurabhdaware/text-to-handwriting/issues/34#issuecomment-656999003

**Comment by Rishika1416** [comment]
we are a team of developers ,we would like to add this feature .Can we proceed?

**Comment by saurabhdaware** [comment]
Yes, sure! Just make sure they are in two different PRs that can be merged separately.

**Comment by Rishika1416** [comment]
yeah! I will make sure to make two different PRs . Will update you soon with any progress.

**Comment by Rishika1416** [comment]
@saurabhdaware Could you please share the acceptance test criteria so that we can work with better efficiency and understand what do you expect after this issue is resolved.

**Comment by saurabhdaware** [comment]
> We can keep everything the way it is right now, but we can also allow users to select text and then change the value in drop-down to color only the selected part.

This ^. So normally when I select ink color, it changes the color of the whole text.

After the PR, 
- if we select a particular text and then change the ink color in dropdown, it will only color the selected part and leave the other part as it is.
- if we don't select any text, then it will change the color of the whole input (the way it works right now)
